### ISSUE TITLE
Introspection error when defining an argument with default value.

### ISSUE THREAD

**Issue by rantingmong** [GraphQLSwift/Graphiti#70]
Given this schema config:

```swift
let schema = try Graphiti.Schema<ResolverRoot, Request> {
    Scalar(UUID.self)

    // define graphql types
    
    Query {
        Field("profile", at: ResolverRoot.profile, as: [Profile].self) {
            Argument("input", at: \.input).defaultValue(.init(page: 0, limit: 10))
        }
    }
    Mutation {
        Field("createProfile", at: ResolverRoot.createProfile, as: Profile.self) {
            Argument("input", at: \.input)
        }
    }
}
```

When asked to be introspected it returns an error:

- On Jetbrains Webstorm:
> A valid schema could not be built using the introspection result. The endpoint may not follow the GraphQL Specification.
> Error: internal error: should never happen: valueFromAst of '{"limit":10,"page":0}' failed because of 'Invalid Syntax : offending token '"limit"' at line 1 column 25'
- On Insomnia
>Failed to fetch schema: Syntax Error: Expected Name, found string "limit"

Removing the `.defaultValue(...)` fixes the problem but runs the impossibility of not having a default value on an optional argument in the query.

#### More context

The input defined is a generic struct:
```swift
struct Input<Payload: Codable>: Codable {
    let input: Payload
}
```

And the payload:
```swift
struct ProfileQueryInput: Codable {
    let page: Int
    var limit: Int = 10
}
```

When defining the GQL types, only `ProfileQueryInput` and not `Input<ProfileQueryInput>` is defined in the schema config.

**Comment by d-exclaimation** [comment]
I couldn't reproduce the issue. From your explanation, nothing seemed out of the ordinary.

I am guess this

```swift
struct Input<Payload: Codable>: Codable {
    let input: Payload
}
```

is for the arguments. 

I tried with a generic Input (Argument structs) and with one without using generic Input. In both cases, all introspection queries are resolved properly. Apollo Sandbox, GraphiQL, and GraphQL Playground have no issue with the introspection result as well. Haven't checked Insomnia and Jetbrains Webstorm, but they shouldn't have issue with the result if GraphiQL doesn't.

The example I used is here
- https://github.com/d-exclaimation/pioneer-example/blob/b5574b02990f9ed0a243037766eab44149bedc3a/Sources/PioneerChatExample/schema.swift#L144
